spec/services/claims/fee_calculator/modifier_decorator_spec.rb

```diff
@@ -24,8 +24,8 @@
   context 'when called with arg of same class of object' do
     let(:other_decorated_modifier) { described_class.new(OpenStruct.new(fixed_percent: '-20.00', percent_per_unit: '0.00')) }
     it 'calls super' do
-      expect(decorated_modifier.send(method, decorated_modifier)).to eql true
-      expect(decorated_modifier.send(method, other_decorated_modifier)).to eql false
+      expect(decorated_modifier.send(method, decorated_modifier)).to be_truthy
```

Comment:
For consistency, can we use expect(decorated_modifier.send(method, other_decorated_modifier)).to be true ?